### PR TITLE
#119 - Fix `time` field in meta.json

### LINKED ISSUE #119: The ‘time’ field is lost during meta.json update. Only the ‘created ’field is displayed.
"time": {
        "1.0.4": "2018-07-25T03:23:08.183Z",
        "1.0.5": "2018-07-25T03:48:17.879Z",
        "1.0.6": "2018-07-25T06:32:03.849Z",
        "1.0.7": "2018-07-25T07:00:53.925Z",
        "1.0.0": "2018-07-24T03:36:13.102Z",
        "1.1.4": "2018-08-22T04:03:22.977Z",
        "1.0.2": "2018-07-24T05:43:35.163Z",
        "1.0.3": "2018-07-25T03:13:39.348Z",
        "1.0.8": "2018-07-25T07:06:46.138Z",
        "1.0.9": "2018-07-25T07:11:44.865Z",
        "1.1.2": "2018-08-15T09:50:45.190Z",
        "1.1.5": "2018-08-22T04:08:19.853Z",
        "1.1.6": "2018-08-22T05:50:17.928Z",
        "1.1.1": "2018-07-25T08:29:09.395Z",
        "1.2.6": "2018-08-24T09:42:13.288Z",
        "1.3.7": "2018-09-10T03:28:14.031Z",
        "1.3.6": "2018-08-29T04:00:08.109Z",
        "1.3.9": "2019-01-22T06:37:50.307Z",
        "1.3.8": "2018-10-27T03:31:00.920Z",
        "1.1.0": "2018-07-25T08:19:19.249Z",
        "created": "2018-07-24T03:36:13.042Z",
        "modified": "2019-03-16T06:25:04.545Z",
        "1.3.13": "2019-03-15T10:22:00.052Z",
        "1.3.12": "2019-03-15T10:15:09.479Z",
        "1.3.11": "2019-01-28T08:30:23.246Z",
        "1.3.10": "2019-01-28T08:26:51.759Z",
        "1.3.14": "2019-03-16T06:25:01.896Z",
        "1.1.3": "2018-08-16T02:32:50.644Z"
    },

### PR DESCRIPTION
**Pull request by genryxy**
Closes #119 
Add time for uploaded version and update `modified` time in `Meta#updateMeta`. Disable test. Also modified `time` field in existing test as it always is contained in `meta.json` because `NpmPublishJsonToMetaSkelethon#skeleton` add them.

### CODE REVIEW

File: src/main/java/com/artipie/npm/Meta.java [hunk 1]
```diff
@@ -105,4 +113,18 @@ public Meta updatedMeta(final JsonObject uploaded) {
...
+     */
+    private static String now() {
+        return DateTimeFormatter.ISO_LOCAL_DATE_TIME
+            .format(
+                ZonedDateTime.ofInstant(
+                    Instant.now(),
+                    ZoneOffset.UTC
+                )
+            );
+    }
```
Comment (olegmoz) [inline src/main/java/com/artipie/npm/Meta.java:129]: @genryxy this code fragment is duplicated, I'd suggest to extract it to separate class with method for generating date/time as string

**Review by olegmoz — CHANGES_REQUESTED** [review]
@genryxy thanks, please see one comment

**Review by olegmoz — APPROVED** [review]
@genryxy thanks

**Comment by 0crat** [comment]
Job `gh:artipie/npm-adapter#134` is not assigned, can't get performer

**Pull request merged**